source/Calamari.Common/Plumbing/Deployment/PackageRetention/TinyType.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Dynamic;
+using YamlDotNet.Core.Tokens;
+
+namespace Calamari.Deployment.PackageRetention
+{
+    //TODO: At some point, replace this basic tiny types implementation with the Octopus one.  The blocking issue is Net40 atm.
+    public abstract class TinyType<T> where T : IComparable
```

Comment:
Another no-op once https://github.com/OctopusDeploy/Calamari/pull/801 is merged.